python/pyarrow/_csv.pyx

```diff
@@ -109,8 +109,8 @@ cdef class ReadOptions:
     @property
     def skip_rows(self):
         """
-        The number of rows to skip at the start of the CSV data, not
-        including the row of column names (if any).
+        The number of rows to skip at the start of the CSV data (the row
+        of column names (if any) does not need to be included in this count).
```

Comment:
Perhaps "The number of rows to skip before the column names (if any) and the CSV data." ?

Reply:
@nealrichardson What do you think?

Reply:
Ah, I like your suggestion (that should also make it clear, without needing my wordy after note)

Reply:
Yeah I like @pitrou's phrasing.